notebooks/ivf_flat_hpo_example.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>The challenge with vector search is that there's a dual objective here- it's not just about the final recall achieved but rather the tradeoff between a "good enough" recall with "good enough" search latency / throughput. The problem with only using recall is that you will usually find the search latency is at its highest. I think we have to consider both in here up front, otherwise this example isn't particularly useful in the practice. Maybe we can accept a target recall try to minimize search latency that reaches at least to the target recall. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuvs/pull/257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>